code/modules/mob/emote.dm

```diff
@@ -32,6 +36,27 @@
 	if(.)
 		user.SpinAnimation(7,1)
 
+/datum/emote/flip/check_cooldown(mob/user, intentional)
+	. = ..()
+	if(.)
+		return
+	if(!can_run_emote(user, intentional=intentional))
+		return
+	if(isliving(user))
+		var/mob/living/flippy_mcgee = user
+		if(prob(20))
+			flippy_mcgee.Knockdown(1 SECONDS)
+			flippy_mcgee.visible_message(
+				"<span class='notice'>[flippy_mcgee] attempts to do a flip and falls over, what a doofus!</span>",
+				"<span class='notice'>You attempt to do a flip while still off balance from the last flip and fall down!</span>"
+			)
+			flippy_mcgee.adjustBruteLoss(rand(0, 1))
```

Comment:
Calling adjustBruteLoss with a 0 as argument feels odd Maybe put an if in front of this.

Reply:
The damage procs handle 0 just fine

Reply:
Just because it works doesn't mean that it's good. We know that this isn't going to do anything if the die lands on 0, so why even call it?

Reply:
Because this is concise and immediately conveys what's happening to the reader without much thought. Adding an if check and var reduces readability a bit and since it doesn't matter here I prefer to value the readability benefit.

Reply:
A prob(50) -> 1 damage is still pretty readable and is still a coin flip

Reply:
Yeah that'd be fine